client/allocrunner/tasklifecycle/doc.go

```diff
@@ -1,3 +1,4 @@
+//nolint
```

Comment:
You can remove this, the formatting was fixed in https://github.com/hashicorp/nomad/pull/14232

Reply:
nice 👍